FernFlower-Patches/0029-Improve-inferred-generic-types.patch

```diff
@@ -463,7 +463,7 @@ index ebb845dff674965ff3e0e8011db33dae5270e24d..a02c40a8a8a2ad1a1888d8a5370625c2
 +
 +          int j = 0;
 +          for (int i = start; i < lstParameters.size(); ++i) {
-+            if (mask == null || mask.get(i) != null) {
++            if (mask == null || mask.get(i) == null) {
```

Comment:
Huh.. wonder how this ever worked. Also probably need to change  isNew to isGenNew a few lines above where the mask is set so that we can make sure desc.getSignature is not null.